lua/auto-session/lib.lua

```diff
@@ -682,6 +682,13 @@ function Lib.get_session_list(sessions_dir)
       path = sessions_dir .. file_name,
     }
   end, entries)
+  
+  -- Filter out nil entries (files that didn't pass is_session_file check)
+  local filtered_result = vim.tbl_filter(function(entry)
+    return entry ~= nil
+  end, result)
+
+  return filtered_result
```

Comment:
and either just return vim.tbl_filter directly or renamed `filtered_result` to `filtered_session_entries`?